pkg/render/manager.go

```diff
@@ -110,21 +110,22 @@ func Manager(cfg *ManagerConfiguration) (Component, error) {
 
 // ManagerConfiguration contains all the config information needed to render the component.
 type ManagerConfiguration struct {
-	KeyValidatorConfig      authentication.KeyValidatorConfig
-	ESSecrets               []*corev1.Secret
-	TrustedCertBundle       certificatemanagement.TrustedBundle
-	ESClusterConfig         *relasticsearch.ClusterConfig
-	TLSKeyPair              certificatemanagement.KeyPairInterface
-	PullSecrets             []*corev1.Secret
-	Openshift               bool
-	Installation            *operatorv1.InstallationSpec
-	ManagementCluster       *operatorv1.ManagementCluster
-	TunnelSecret            certificatemanagement.KeyPairInterface
-	InternalTrafficSecret   certificatemanagement.KeyPairInterface
-	ClusterDomain           string
-	ESLicenseType           ElasticsearchLicenseType
-	Replicas                *int32
-	ComplianceFeatureActive bool
+	KeyValidatorConfig             authentication.KeyValidatorConfig
+	ESSecrets                      []*corev1.Secret
+	TrustedCertBundle              certificatemanagement.TrustedBundle
+	ESClusterConfig                *relasticsearch.ClusterConfig
+	TLSKeyPair                     certificatemanagement.KeyPairInterface
+	PullSecrets                    []*corev1.Secret
+	Openshift                      bool
+	Installation                   *operatorv1.InstallationSpec
+	ManagementCluster              *operatorv1.ManagementCluster
+	TunnelSecret                   certificatemanagement.KeyPairInterface
+	InternalTrafficSecret          certificatemanagement.KeyPairInterface
+	ClusterDomain                  string
+	ESLicenseType                  ElasticsearchLicenseType
+	Replicas                       *int32
+	Compliance                     *operatorv1.Compliance
+	ComplianceLicenseFeatureActive bool
```

Comment:
I don't think `Feature` adds any information to this variable name with `License` in the name now so please remove it since it is a bit long.

Reply:
My original intent was to differentiate between license feature and product feature, however I was on the fence. I can dock `License`

Reply:
I was trying to suggest we remove `Feature` from the name if we're including `License`.

Reply:
:facepalm: